tools/on_device_tests_gateway_client.py

```diff
@@ -90,6 +81,24 @@ def run_command(self, workdir: str, args: argparse.Namespace):
             build_number=args.build_number,
             loader_platform=args.loader_platform,
             loader_config=args.loader_config,
+            version=args.version,
+            dry_run=args.dry_run,
+            dimension=args.dimension or [],
```

Comment:
The default value for the `dimension` is empty list. I'm treating it as a `string` in the `on_device_tests_gateway_client.py` to keep things simple, this is why I'm passing an empty list here in case tests don't need a dimension. Also, same behavior exists in buildbot: `dimensions = args.dimension or []`